controllers/backplaneconfig_controller.go

```diff
@@ -327,6 +332,14 @@ func (r *MultiClusterEngineReconciler) SetupWithManager(mgr ctrl.Manager) error
 				}
 			},
 		}, builder.WithPredicates(predicate.LabelChangedPredicate{})).
+		Watches(&source.Kind{Type: &configv1.ClusterVersion{}}, &handler.Funcs{
```

Comment:
Can you explain how this watch works?

Reply:
This checks if the openshift ClusterVersion struct updates in any way. The item we're watching for in that struct is `ClusterVersionStatus.ClusterVersionCapabilitiesStatus` which will change if Console is added to the Enabled Capabilities in OCP 4.12+